Orso.Arpa.Domain/Logic/SelectValues/List.cs

```diff
@@ -36,7 +36,8 @@ public Task<IImmutableList<SelectValueMapping>> Handle(Query request, Cancellati
                         && c.Property.ToLower() == request.PropertyName.ToLower())
 #pragma warning restore RCS1155 // Use StringComparison when comparing strings.
                     .SelectMany(c => c.SelectValueMappings)
-                    .OrderBy(s => s.SelectValue.Name)
+                    .OrderBy(s => s.SortOrder)
```

Comment:
Sort Order ist not nullable und wird mit dem Defaultwert 0 initialisiert. Mit der Sortierlogik hier würden die Objekte mit SortOrder == 0 an den Anfang der Liste gestellt werden. Dies würde auch mit NULL passieren, wenn es sich um einen nullable int handeln würde. Allerdings halte ich es dennoch für sinnvoller, SortOrder nullable zu machen und mit NULL vorzubelegen. Warum sollte man nicht auch 0 als gewollten Reihenfolgewert eingeben können? Im Fall von einem nullable int würde folgendes Snippet korrekt sortieren: `.OrderBy(s=> s.SortOrder ?? int.MaxValue)`

Reply:
Mir fällt gerade ein: Die Sortierung nach Name macht hier nur bedingt Sinn, weil die Texte u.U. ja nachträglich nochmal übersetzt werden

Reply:
nullable: ich hatte im SelectValueMapping eine Vorbelegung mit maxInt drin. Hatte ebenfalls funktioniert aber natürlich in der Migration für alle SelectValueMappings einen Eintrag gemacht. Den nullable int noch eine Ebene weiter runter in die Datenbank zu bringen macht es eleganter.

Übersetzung: ja klar. Da kann es sein, dass im anderssprachigen Setup die Reihenfolge anders ist. Wenn man mit alphabetisch sortierten Texten arbeitet, ist das IMHO ok. Möchte man das nicht, kann man ja mittels SortOrder eine Vorgabe machen.